apps/documentation/src/routes/documentation/hooks/use-router.mdx

```diff
@@ -35,7 +37,7 @@ export default function IndexPage(): JSX.Element {
 }
 ```
 
-## `useRouter` result
```

Comment:
This section describes the returned  value of the `useRouter`, IMHO result is more semantic.

Using object make useRouter look like an object rather than a hook